saas/forms.py

```diff
@@ -208,15 +208,16 @@ def __init__(self, *args, **kwargs):
             if not self.fields['country'].initial:
                 self.fields['country'].initial = country.code
             self.add_postal_region(country=country)
-        if 'is_bulk_buyer' in self.initial:
-            initial = self.initial['is_bulk_buyer']
-            if self.instance:
-                initial = self.instance.is_bulk_buyer
-            self.fields['is_bulk_buyer'] = forms.BooleanField(required=False,
-                initial=initial,
-                label=mark_safe(_("Enable GroupBuy (<a href=\""\
-"https://djaodjin.com/docs/#group-billing\" target=\"_blank\">what is it?</a>)"
-                )))
+        if settings.DISPLAY_GROUP_BUY:
+            if 'is_bulk_buyer' in self.initial:
```

Comment:
This `if 'is_bulk_buyer' in self.initial` condition to add the form field or not makes me think the best place to add the `if settings.DISPLAY_GROUP_BUY` is in `saas.views.profile.OrganizationProfileView.get_initial`.